nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -27,7 +27,7 @@
 def load_dataset_with_features(
     dataset="all_baseline",
     remove_growth_outliers=True,
-    load_local=False,
+    load_local=True,
```

Comment:
```suggestion
    load_local=False,
```
Default to not load local would be best!